ui/app/pages/send/send.component.js

```diff
@@ -161,12 +161,19 @@ export default class SendTransactionScreen extends Component {
     if (qrCodeData) {
       if (qrCodeData.type === 'address') {
         scannedAddress = qrCodeData.values.address.toLowerCase()
-        const currentAddress = prevTo?.toLowerCase()
-        if (currentAddress !== scannedAddress) {
-          updateSendTo(scannedAddress)
-          updateGas = true
-          // Clean up QR code data after handling
+        const toError = getToErrorObject(scannedAddress, false, network)
```

Comment:
Wow, that is an.... interesting method name and signature :grimacing:. It does the trick though at least!

Reply:
Though actually, it does give a pretty strange error on testnets :thinking: 

`Not ETH network, set to lowercase` seems inappropriate here

Reply:
It might be worth using `isValidAddress` and `INVALID_RECIPIENT_ADDRESS_ERROR` directly perhaps, instead of using that rather confusing helper function

Reply:
@Gudahtt -- I was following the pattern in the file and had it working before I realized how... wonky it was. 🤦‍♂️